stdlib/public/core/StringUTF16View.swift

```diff
@@ -480,7 +483,7 @@ extension String.UTF16View {
   internal func _nativeGetIndex(for offset: Int) -> Index {
     // Trivial and common: start
     if offset == 0 { return startIndex }
-
+    
```

Comment:
I had an ASCII fast path here too but I got it wrong and it was crashing. Need to debug further.